src/ripple/app/ledger/impl/LedgerCleaner.cpp

```diff
@@ -301,7 +311,8 @@ class LedgerCleanerImp
 
         if(! getApp().getLedgerMaster().fixIndex(ledgerIndex, ledgerHash))
         {
-            m_journal.debug << "ledger " << ledgerIndex << " had wrong entry in history";
+            m_journal.debug << "ledger " << ledgerIndex
+                            << " had wrong entry in history";
```

Comment:
I find the leading `<<` on the continuation line quite readable.  Though I don't know if we want to dictate style down to this degree.  I'm in favor of the official style promoting readable code, as opposed to where to place `<<` in a situation like this.